src/qililab/calibration/calibration_node.py

```diff
@@ -549,16 +554,17 @@ def _create_notebook_datetime_path(
 
         # If doesn't exist, create the needed folder for the path
         os.makedirs(self.nb_folder, exist_ok=True)
+        abs_path = os.path.abspath(self.nb_folder)
```

Comment:
Add in the docstring, that returns an absolute path, maybe?

Reply:
Added